frontend/src/layout/onboarding/onboarding.js

```diff
@@ -0,0 +1,162 @@
+import { userLogic } from 'scenes/userLogic'
+import _ from 'lodash'
+import api from 'lib/api'
+
+export function OnboardingWidget() {
+    const contentRef = useRef()
+    const { actions, actionsLoading } = useValues(actionsModel)
+    const [instructionalModal, setInstructionalModal] = useState(false)
+    const { user } = useValues(userLogic)
+    const { loadUser } = useActions(userLogic)
+    const { tourType, checked } = useValues(onboardingLogic({ user }))
+    const { setTourActive, setTourType, updateOnboardingInitial } = useActions(onboardingLogic)
+    const [visible, setVisible] = useState(user.onboarding.initial ? true : false)
+
+    const unfinishedCount = _.filter(checked, (isChecked) => !isChecked).length
+
+    function closePopup() {
+        if (user.onboarding.initial) updateOnboardingInitial(false)
+        setVisible(false)
+    }
+
+    let onClickOutside = (event) => {
+        if (contentRef.current && !contentRef.current.contains(event.target)) {
+            closePopup()
+        }
+    }
+
+    useEffect(() => {
+        document.addEventListener('mousedown', onClickOutside)
+        return () => {
+            document.removeEventListener('mousedown', onClickOutside)
+        }
+    }, [])
+
+    async function dontShowAgain() {
+        await api.update('api/user', { onboarding: { ...user.onboarding, active: false } })
+        loadUser()
+    }
+
+    function content() {
+        return (
+            <div
+                data-attr="onboarding-content"
+                ref={contentRef}
+                style={{ display: 'flex', width: '25vw', flexDirection: 'column' }}
+            >
+                <h2>Get Started</h2>
+                <i>
+                    Complete these steps to learn how to use Posthog! Click on an item below to learn how to create one
+                </i>
+                {Object.entries(TourType).map(([, value], index) => {
+                    return (
+                        <div key={index}>
+                            <hr style={{ height: 3, visibility: 'hidden' }} />
+                            <Link
+                                onClick={() => {
+                                    closePopup()
+                                    setInstructionalModal(true)
+                                    setTourType(value)
+                                }}
+                                data-attr={'onboarding-item-' + index}
+                            >
+                                <Checkbox
+                                    style={{ marginRight: 12 }}
+                                    checked={user.onboarding.steps[index] || checked[index]}
+                                ></Checkbox>
+                                <span>{value}</span>
+                            </Link>
+                        </div>
+                    )
+                })}
+                <hr style={{ height: 5, visibility: 'hidden' }} />
+                {unfinishedCount > 0 ? (
```

Comment:
Consider using `!== 0` instead of `> 0` for parity with the inverse:
```
const isArrayEmpty = array.length === 0;
const hasItems = array.length !== 0;
```
(src: _Code Complete 2nd Edition_ but it's a pretty common guideline)